book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  attr_reader :rentals
+
+  def initialize(author:, title:)
+    @author = author
+    @title = title
+    @rentals = []
+  end
+
+  attr_accessor :author, :tittle
+end
```

Comment:
- [ ] As part of the requirements, you are supposed to create a has-many relationship side of Book and Rental. So kindly add a method to add rentals in the book Class